data/categories.txt

```diff
@@ -5920,7 +5920,7 @@ da:^Skomager|skoreparation
 nl:^Schoenmaker|schoenmakerij
 fi:^Suutari
 fr:^Cordonnier
-de:^Schuhreparatur
+de:^Schuhreparatur|Schuster
```

Comment:
Ist Schuster länderspezifisch? 
Hab's nie im Norden oder Ostern Deutschlands gehoert

Reply:
Ob es länderspezifisch ist, kann ich nicht sagen. Meine [Namensauswertung](https://overpass-turbo.eu/s/J5R) für Deutschland hat folgendes ergeben:
Schuhmacher 141
Schuster 18
Schuhreparatur 47

Ich schlage daher "Schuhmacher" vor und würde zusätzlich "Schuster" in categories.txt aufnehmen.

Reply:
> Ich schlage daher "Schuhmacher" vor und würde zusätzlich "Schuster" in categories.txt aufnehmen.

Ja, so ist gut. Danke!